bot/cogs/bot.py

```diff
@@ -342,7 +343,11 @@ def has_bad_ticks(self, msg: Message):
                         howto_embed = Embed(description=howto)
                         bot_message = await msg.channel.send(f"Hey {msg.author.mention}!", embed=howto_embed)
                         self.codeblock_message_ids[msg.id] = bot_message.id
-                        await bot_message.add_reaction(Emojis.cross_mark)
+
+                        # Create
```

Comment:
Did you forget to type this out?

Reply:
I think the code is adequately self-descriptive anyway.

```suggestion
```

Reply:
agreed

Reply:
and yeah oops I guess I forgot to write it